superset/config.py

```diff
@@ -1592,7 +1592,7 @@ def EMAIL_HEADER_MUTATOR(  # pylint: disable=invalid-name,unused-argument  # noq
 TALISMAN_CONFIG = {
     "content_security_policy": {
         "base-uri": ["'self'"],
-        "default-src": ["'self'"],
+        "default-src": ["'self'", "data:"],
```

Comment:
When you include `data:` in your CSP's `default-src` or any other directive, it allows any data URI to be loaded, regardless of its source. This means that any data URL can be executed or rendered by the browser, which can pose security risks such as exposing your application to cross-site scripting (XSS) attacks.

To restrict sources to trusted domains, you can specify the domains explicitly in the `default-src` or other specific directives (like `script-src`, `style-src`, etc.)

```
Content-Security-Policy: default-src 'self' https://trusted.com; 
```

Reply:
update to font-src

Reply:
@justinpark Can you restrict `font-src` to trusted domains?